addons/nametags/config.cpp

```diff
@@ -13,7 +13,8 @@ class CfgPatches {
 };
 
 #include "CfgEventHandlers.hpp"
-#include "ACE_Settings.hpp"
+#include "CBA_Settings.hpp"
```

Comment:
Mybe we should not change the file name, which would make the PR much easier to review